.github/workflows/operator-regression.yml

```diff
@@ -23,6 +27,33 @@ jobs:
         run:
            bash test/operator/kind-with-registry.sh
 
+      - name: Start Elasticsearch
+        run: |
+           docker network create elastic
+           #Local elasticsearch without SSL on http://localhost:9200
+           docker run --name es01 --net elastic -p 9200:9200 -m 1GB -e xpack.security.enabled=false -e xpack.security.enrollment.enabled=false -e discovery.type=single-node docker.elastic.co/elasticsearch/elasticsearch:8.16.1 1>es.log 2>&1 &
+           bash test/operator/utilities/wait_for_es_http_ready.bash http://localhost:9200
+
+      - name: Start Collector
+        run: |
+           #Elastic agent from download instructions
+           curl -L -O https://artifacts.elastic.co/downloads/beats/elastic-agent/elastic-agent-8.16.1-linux-x86_64.tar.gz
+           tar xzvf elastic-agent-*-linux-x86_64.tar.gz
+           cd elastic-agent-*-linux-x86_64/
+           #Use latext collector config for elastic agent
+           curl -L -O https://raw.githubusercontent.com/elastic/elastic-agent/refs/heads/main/internal/pkg/otel/samples/linux/logs_metrics_traces.yml
+           #bind to ANY rather than LOOPBACK so that the pods can connect
+           sed -e 's/http:/http:\n        endpoint: 0.0.0.0:4318/' logs_metrics_traces.yml > logs_metrics_traces2.yml
```

Comment:
If you don't need a separate untouched file you can pass `-i` to sed to change the file in place

Reply:
I wanted to make a copy here, it's useful when debugging to have the original and the copy